gitlab/client.py

```diff
@@ -16,6 +17,8 @@
 import gitlab.exceptions
 from gitlab import utils
 
+LOG = logging.getLogger(__name__)
```

Comment:
Since it is a global I prefer the `LOG`. Also in the past I have used `logger` but then found myself accidentally typing `logging` and not noticing what went wrong for awhile :(

Plus it is the way OpenStack does it:
https://github.com/openstack/ironic/blob/0659485d630b8651faa633f98b1802bdf244f186/ironic/drivers/modules/ipmitool.py#L61

Reply:
As an example it is easy to do:

`logging.debug("some message")` when you really meant to do `logger.debug("some message")`

Reply:
Ok, it was just an idea as it is a [Logger](https://docs.python.org/3/library/logging.html#logging.Logger) instance that we manipulate and not a constant as `LOG` would imply.  I guess `log` would also be somewhere in between.

Reply:
What do you think of the name `LOGGER`? For me it is the middle ground.